_data/meltano/utilities/airflow/meltano.yml

```diff
@@ -0,0 +1,81 @@
+name: airflow
+namespace: airflow
+label: Airflow
+docs: https://docs.meltano.com/guide/orchestration
+repo: https://github.com/meltano/airflow-ext
+pip_url: git+https://github.com/meltano/airflow-ext.git@main apache-airflow==2.3.3 --constraint https://raw.githubusercontent.com/apache/airflow/constraints-2.3.3/constraints-no-providers-3.8.txt
+executable: airflow_invoker
+variant: meltano
+maintenance_status: active
+commands:
```

Comment:
Can you add in the commands from https://hub.meltano.com/orchestrators/airflow#commands ?

Reply:
👍 yep let me bring them over. I assumed we didn't need those anymore but in hindsight makes sense that we'd keep them around.